packages/check-html-links/package.json

```diff
@@ -34,6 +34,7 @@
   ],
   "dependencies": {
     "chalk": "^4.0.0",
+    "check-links": "^1.1.8",
```

Comment:
hmmm check-links seems to be a tiny thing... 😅

https://github.com/transitive-bullshit/check-links/blob/master/lib/check-link.js

could we maybe convert it to esm, using fetch and `new URL` instead?
I really wanna keep the dependencies small 🤔

Reply:
I take a dependency-free approach. Take a look and tell me what do you think.

But I'm having issue with test. Cleanup is not working or something related with async. I can't find the error.